packages/ui-components/src/components/DataTable/SQLQueryEditor.js

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Works great nice one!

Were you able to get error highlighting going?

Reply:
Good point, seems like we can only identify error, but not point out from where. Still worth to add?

Reply:
I think it's worth spending a couple hours looking into it, if too hard we can make a card for it for the future.

Reply:
Cool, I will try to add it.

Reply:
I've found a hacky way to do it, please see commit https://github.com/beyondessential/tupaia/pull/4312/commits/6adf8d6aa26f2c2e6d486d46573b4208e0c73cf6